opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -144,9 +146,13 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
-class Span:
+class Span(object):
     """A span represents a single operation within a trace."""
 
+    def __init__(self, auto_update_status=False):
```

Comment:
I don't really like adding constructor arguments here. Maybe we move the behavior controlled by this into the SDK?

Reply:
Ok, I moved it. Please let me know if this is what you were expecting.